docs/annotations.md

```diff
@@ -0,0 +1,105 @@
+# Manifest Annotations
+
+[Annotations](<https://github.com/opencontainers/image-spec/blob/master/annotations.md>) are supported by [OCI Image Manifest](<https://github.com/opencontainers/image-spec/blob/master/manifest.md#image-manifest>) and [OCI Content Descriptors](<https://github.com/opencontainers/image-spec/blob/master/descriptor.md>) so as `oras`.
```

Comment:
"so as oras" sounds a little strange.. maybe something like "Annotations, which are supported by ... and .., are also supported by oras"